### PR TITLE
Added safety definition

### PR DESCRIPTION
**Pull request by zhongjun2**

### CODE REVIEW

**Comment by germonprez** [comment]
Hi @zhongjun2 

Thanks for the PR. A few comments: 

1) Could you add a DCO sign-off for the PR
2) Could you place the markdown file in a /definition folder 
3) I think you just include the metric and link to the metric in the Metrics in the Model section

**Comment by zhongjun2** [comment]
> Hi @zhongjun2
> 
> Thanks for the PR. A few comments:
> 
> 1. Could you add a DCO sign-off for the PR
> 2. Could you place the markdown file in a /definition folder
> 3. I think you just include the metric and link to the metric in the Metrics in the Model section

Done

> Hi @zhongjun2
> 
> Thanks for the PR. A few comments:
> 
> 1. Could you add a DCO sign-off for the PR
> 2. Could you place the markdown file in a /definition folder
> 3. I think you just include the metric and link to the metric in the Metrics in the Model section

Done

**Comment by eyehwan** [comment]
@zhongjun2 Does this model include data insight?

**Comment by zhongjun2** [comment]
> @zhongjun2 Does this model include data insight?

The document does not yet present the data and associated analysis

File: metrics-model-libs/safety/definition/safety.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+|Difficulty: Medium|
+|Audience: Community Manager/Maintainer, OSPO|
+|Description: One symptom of burnout can be lack of responsiveness to issues, comments, PRs , discussion…  It may not be the only, or primary reason but is a key indicator of risk with regard to expectations of responsiveness to issues related to safety.|
+
+
+
+# Data Insights 
+
+## Background of the Investigated Context
+We choose a Linux OS distribution community to show metric model.
```
Comment (eyehwan) [inline metrics-model-libs/safety/definition/safety.md:186]: What does this mean?

File: metrics-model-libs/safety/definition/safety.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+
+
+# Data Insights 
+
+## Background of the Investigated Context
+We choose a Linux OS distribution community to show metric model. 
+
+## Insights Drawn from the Metrics Model
+Provide text and visualizations that describe how the unified metrics model is used to draw insights from the evaluated community or communities.
```
Comment (eyehwan) [inline metrics-model-libs/safety/definition/safety.md:189]: if it doesn't privde insight, I think we should move it.

File: metrics-model-libs/safety/definition/safety.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+We choose a Linux OS distribution community to show metric model. 
+
+## Insights Drawn from the Metrics Model
+Provide text and visualizations that describe how the unified metrics model is used to draw insights from the evaluated community or communities. 
+
+## Metric Specific Insights Drawn from the Metrics Model
+
+
+# References
+https://docs.google.com/document/d/1DpHhVMAwucaKoXoOYMCnmxZuhWMqWupDsZvWpxe_A0M/edit
```
Comment (eyehwan) [inline metrics-model-libs/safety/definition/safety.md:195]: this is the draft for review, not references. Papers, books, articles are suitable here.

**Comment by germonprez** [comment]
Hi @zhongjun2 -- Thanks for continuing to move this forward. A few more suggestions: 

1) Could you simply provide a link to the metric used in the metric model? I don't think that we need the rich text below the link to the metric. 
2) You could likely follow the suggestions from @eyehwan to remove the headers that are not being used. We can return to include them when we have a sample community to talk about.

File: metrics-model-libs/safety/definition/safety.md (hunk 1 above)
Comment (zhongjun2) [reply metrics-model-libs/safety/definition/safety.md]: done

File: metrics-model-libs/safety/definition/safety.md (hunk 3 above)
Comment (zhongjun2) [reply metrics-model-libs/safety/definition/safety.md]: done

**Comment by zhongjun2** [comment]
> Hi @zhongjun2 -- Thanks for continuing to move this forward. A few more suggestions:
> 
> 1. Could you simply provide a link to the metric used in the metric model? I don't think that we need the rich text below the link to the metric.
> 2. You could likely follow the suggestions from @eyehwan to remove the headers that are not being used. We can return to include them when we have a sample community to talk about.

Added link to the metrics in the doc. Removed the headers that are not being used

**Review by sgoggins — APPROVED** [review]
LGTM

**Comment by eyehwan** [comment]
Looks good to me!

**Pull request merged**